docs/user/dql/functions.rst

```diff
@@ -1762,6 +1762,28 @@ Example::
     | 2                           |
     +-----------------------------+
 
+MINUTE_OF_DAY
+------
+
+Description
+>>>>>>>>>>>
+
+Usage: minute_of_day(time) returns the minute value for time within a 24 hour day, in the range 0 to 1439.
+
+Argument type: STRING/TIME/DATETIME/TIMESTAMP
+
+Return type: INTEGER
+
+Example::
+
+    os> SELECT MINUTE_OF_DAY((TIME '01:02:03'))
```

Comment:
why double bracket?

Reply:
There was no specific reason. I copied the test used for the `minute` function because this function works in a similar way, and it already had the double brackets used in the doctest.